apps/web_delivery/mix.exs

```diff
@@ -36,4 +31,11 @@ defmodule WebDelivery.Mixfile do
       {:rollbax, "~> 0.8.0"},
     ]
   end
+
+  defp elixirc_paths(:test), do: ["lib", "web", "test/support"]
+  defp elixirc_paths(_),     do: ["lib", "web"]
+
+  def application do
+    [mod: {WebDelivery.Application, []}, extra_applications: [:logger]]
+  end
```

Comment:
It’s all relative. 😆 Actually, `deps/0` and `elixirc_paths/1` were moved directly under `project/0` since they are private functions only used by it.

> If a private function is used by only one public function, place it directly below that function. If a private function is used by multiple public functions, place it below the the last function that uses it.

It is the organizational equivalent of lexical scoping. It’s especially helpful when your syntax highlighting is set to differentiate public and private functions from one another, like so:
<img width="507" alt="public/private differentiation" src="https://cloud.githubusercontent.com/assets/2033828/22650344/ce0ec0c8-ec4c-11e6-8f9a-a728b3bf309f.png">